apps/common/test/lexical/ast_test.exs

```diff
@@ -255,6 +255,23 @@ defmodule Lexical.AstTest do
     end
   end
 
+  describe "expand_aliases/4" do
+    test "works with __MODULE__ aliases" do
+      {position, document} =
+        ~q[
+        defmodule Parent do
+          defmodule __MODULE__.Child do
+            |
+          end
+        end
+      ]
+        |> pop_cursor(as: :document)
```

Comment:
These are equivalent, but I personally much prefer how this looks with the bracket lined up with the following pipe:

```suggestion
        ~q[
          defmodule Parent do
            defmodule __MODULE__.Child do
              |
            end
          end
        ]
        |> pop_cursor(as: :document)
```

I also think it's worth it just to be consistent -- the other two tests that were added were formatted like this.